packages/taquito-utils/src/validators.ts

```diff
@@ -81,7 +84,7 @@ const blockPrefix = [Prefix.B]
  * // This example return 3 which correspond to VALID
  * ```
  */
-export function validateAddress(value: any): ValidationResult {
+export function validateAddress(value: string | string[]): ValidationResult {
```

Comment:
I think this should only by string, no array

Reply:
The test case `expect(validateAddress([])).toEqual(ValidationResult.NO_PREFIX_MATCHED);` doesn't make really sense and can be removed.